source/extensions/common/wasm/context.cc

```diff
@@ -309,12 +309,12 @@ WasmResult serializeValue(Filters::Common::Expr::CelValue value, std::string* re
     return WasmResult::Ok;
   case CelValue::Type::kInt64: {
     auto out = value.Int64OrDie();
-    result->assign(reinterpret_cast<const char*>(&out), sizeof(int64_t));
+    result->assign(std::to_string(out));
     return WasmResult::Ok;
   }
   case CelValue::Type::kUint64: {
     auto out = value.Uint64OrDie();
-    result->assign(reinterpret_cast<const char*>(&out), sizeof(uint64_t));
+    result->assign(std::to_string(out));
```

Comment:
I don’t think this is the correct fix.
result std::string is only used as a container of bytes.
The to_string needs to be applied on the receiving side inside the vm.

Reply:
It is only labels that need everything as a string, if the extension is adding numbers it would have to change format again.

Reply:
but we don't know on extensions side whether the return type is a string or not right? should we change evaluateexpression to return that?